### PR TITLE
Make JFR compile on z/OS

### PR DESCRIPTION
**Pull request by thallium**
- ->system seems to be replaced by the compiler
- = {0} initialization isn't supported by the compiler

### CODE REVIEW

**Comment by thallium** [comment]
@tajila FYI

File: runtime/vm/jfr.cpp [hunk 1]
```diff
@@ -939,12 +939,15 @@ jfrThreadCPULoad(J9VMThread *currentThread, J9VMThread *sampleThread)
...
-				jfrEvent->user = 0;
-				jfrEvent->system = 0;
+				jfrEvent->userCPULoad = 0;
+				jfrEvent->systemCPULoad = 0;
 			} else {
 				int64_t timeDelta = currentTime - jfrState->prevTimestamp;
-				jfrEvent->user = OMR_MIN((threadTimes.userTime - jfrState->prevThreadCPUTimes.userTime) / (double)timeDelta, 1.0);
-				jfrEvent->system = OMR_MIN((threadTimes.sysTime - jfrState->prevThreadCPUTimes.sysTime) / (double)timeDelta, 1.0);
+				jfrEvent->userCPULoad = OMR_MIN((threadTimes.userTime - jfrState->prevThreadCPUTimes.userTime) / (double)timeDelta, 1.0);
+#if !defined(J9ZOS390)
```
Comment (tajila) [inline runtime/vm/jfr.cpp:947]: is this ifdef still needed?

**Comment by thallium** [comment]
https://hyc-runtimes-jenkins.swg-devops.com/job/Build_JDK11_s390x_zos_Personal/4259/ built successfully on zos

**Comment by tajila** [comment]
jenkins test sanity.functional xlinux jdk17

**Comment by tajila** [comment]
please make a 0.49 PR

**Pull request merged**